src/components/themed/AccountInfo.tsx

```diff
@@ -124,13 +121,13 @@ export const AccountInfo = (props: Props) => {
         </Animated.View>
       </TouchableWithoutFeedback>
       <Animated.View style={[styles.info, infoStyle]}>
-        {infoRows.map(({ detail, label }: InfoRow, index: number) =>
-          renderInfoRow(detail, label, index + 1 === infoRows.length)
+        {infoRows.map(({ value, label }: InfoRow, index: number) =>
```

Comment:
I feel like this destructuring is backwards. Everywhere else it's `{ label, value }`.